ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -116,6 +116,7 @@ public enum ControlTokenValue {
     case dynamicColor(() -> DynamicColor)
     case fontInfo(() -> FontInfo)
     case shadowInfo(() -> ShadowInfo)
+    case buttonDynamicColors(() -> ButtonDynamicColors)
 
```

Comment:
Ah, @mischreiber just commented on my button PR for this one, but turns out we're not supposed to add this. Not sure why it's still around in the fluent2-tokens branch if it isn't supposed to make it to main, though.